2/Dockerfile

```diff
@@ -61,9 +61,9 @@ ENV M2 $M2_HOME/bin
 ENV PATH $M2:$PATH
 
 # nodejs
-ENV NODE_VERSION 8.7.0
-RUN wget http://nodejs.org/dist/v0.10.30/node-v0.10.30-linux-x64.tar.gz
-RUN tar --strip-components 1 -xzvf node-v* -C /usr/local
+ENV NODE_VERSION 8.9.4
```

Comment:
- doesn't the commit say 8.7.0 ? I am happy to accept 8.9 since it is LTS but all f8 projects use 8.3.0 but you may want to ask the ui folks
- I think we must verify the checksum after download
- also you may want to combine all the steps under one `RUN` to save some image space.